codespell_lib/data/dictionary.txt

```diff
@@ -12806,9 +12808,13 @@ plugable->pluggable
 pluign->plugin
 pluigns->plugins
 pluse->pulse
+plyotropy->pleiotropy
+pneumonic->mnemonic
```

Comment:
Also suggest pneumonia.

Reply:
Done

Reply:
Come one friends! You cannot always add valid words to codespell.
https://en.wikipedia.org/wiki/Pneumonic

Reply:
In the context of the text that was corrected, it was OK replace "pneumonic" with "mnemonic" (maybe computer science). But in a medical context, it would be exactly the other way around.

Reply:
Removed.

@arm-in: we are waiting for your perfect patches. You can review the dictionary and submit.

Reply:
> Come one friends! You cannot always add valid words to codespell.
> https://en.wikipedia.org/wiki/Pneumonic

Oops, mea culpa @arm-in . What's odd is having looked it up, pleiotropy is a medical term.

Reply:
> @arm-in: we are waiting for your perfect patches. You can review the dictionary and submit.

I've opened https://github.com/codespell-project/codespell/issues/1140 to track getting Travis to try and catch this sort of thing for us in future.

Reply:
In my scripts, before to insert a couple "a->b" in my list, I check them against the usual historycal dictionaries you can find on Unix (like  /usr/share/dict/american-english). But of course, there are a lot of specific (and not so specific...) words not included.

Reply:
> @arm-in: we are waiting for your perfect patches. You can review the dictionary and submit.

@Gelma Please forgive me for being obtrusive.
No, my patches are not always perfect. I'm not a native speaker either.
And yes, I highly appreciate that you contribute an enormous number of entries.
But I think high quality is needed to maintain a dictionary. In many cases where I'm in doubt, I ask Google for enlightenment. I have to admit that this is cumbersome. Maybe @peternewman found a good solution with #1140 to reduce noise.

Reply:
These rare ones can go in https://github.com/codespell-project/codespell/blob/master/codespell_lib/data/dictionary_rare.txt in future.